### DISCUSSION TITLE
230719 데모데이 구상 회의 (10:40 ~ 11:25)

### DISCUSSION THREAD

**Discussion by Cyma-s** [woowacourse-teams/2023-shook#83] [회의]
## 회의 안건
- 데모를 어떻게 보여줄 지 결정한다.
- 데모데이 역할 분담, 데모데이 자료 취합

## 예상 회의 시간
1~2시간

**Comment by splitCoding** [comment]
https://github.com/woowacourse-teams/2023-shook/discussions/62
